app/controllers/journeys/abroad/SelfEmploymentAbroadCYAController.scala

```diff
@@ -50,9 +50,9 @@ class SelfEmploymentAbroadCYAController @Inject()(override val messagesApi: Mess
     Ok(view(taxYear, summaryList, nextRoute, isAgent))
   }
 
-  private def nextPageUrl(taxYear: Int, optBusinessId: String, navigator: Navigator)(implicit request: DataRequest[AnyContent]): String =
+  private def nextPageUrl(taxYear: Int, optBusinessId: String, navigator: AbroadNavigator)(implicit request: DataRequest[AnyContent]): String =
```

Comment:
BusinessID is no longer optional in the navigator, this should be renamed from `optBusinessId` to `businessId`